Makefile

```diff
@@ -0,0 +1,34 @@
+
+build: fetch
+	xcodebuild -project ld/zld.xcodeproj -scheme zld -derivedDataPath build -configuration Release build
+
+abseil-cpp-20200225:
```

Comment:
note that if this fails halfway through, and leaves the abseil folder but not libabsl.a, it will consider the phase done. but it's fine for now